app/packs/src/apps/mydb/elements/details/researchPlans/researchPlanTab/ImageAttachmentFilter.js

```diff
@@ -0,0 +1,11 @@
+export default class ImageAttachmentFilter {
```

Comment:
This is a utility class and should be placed in /packs/src/utilities I think.
The /packs/src/app/ and /packs/src/components/ folders should only contain react components if possible